script/configs/exclude_xcode_deprecation.yaml

```diff
@@ -0,0 +1,2 @@
+# TODO(louisehsu): Remove deprecation check when StoreKit 2 is adopted. https://github.com/flutter/flutter/issues/116383
```

Comment:
@LouiseHsu sorry to assign you a TODO, but this should be good to remove once you have the StoreKit 1 deprecation stuff migrated.
This file and the `--exclude=` part of the target args can be removed.

Reply:
np sounds good!